scripts/getColors.js

```diff
@@ -1,34 +1,40 @@
-const { checkColor } = require('./paletteUtils');
+const { checkColors } = require('./paletteUtils');
 const { URL, TABLE_WIDTH } = require('../constants');
 
 
-function getColors(arr = []) {
-    return new Promise((resolve, reject) => {
-        fetch('url-url-url')
-        // I want to use async await instead of a lot of thens
-        .then(res => res.json())
-        .then((res) => {
-            const colors = [...arr];
-            res.colors.forEach(element => {
-                if (checkColor(element.hex) && element.hex !== 'open') {
-                    colors.push(element.hex)
-                }
-            });
-            return colors;
-        })
-        .then((res) => {
-            if(res.length < TABLE_WIDTH) {
-                console.log('not enought colors! only:' + res.length);
-                res(getColors(resolve));
-                resolve(getColors(res));
-            } else {
-                res.splice(TABLE_WIDTH);
-                console.log('Enougth! ' + res)
-                resolve(res);
-            }
-        })
-        .catch((err) => reject(new Error(err)));
-    })
-}
+const getColors = async (arr = []) => {
+  const data = await getData(URL);
+  const colors = [...arr,...checkColors(data)];
```

Comment:
need space after comma

Reply:
checkColors isn't suit name for this method, it need called like getCheckedColors